src/main/scala/mesosphere/marathon/tasks/PortsMatcher.scala

```diff
@@ -29,7 +29,7 @@ case class PortsMatch(hostPortsWithRole: Seq[Option[PortWithRole]]) {
 /**
   * Utility class for checking if the ports resource in an offer matches the requirements of an app.
   */
-class PortsMatcher(
+class PortsMatcher private[tasks] (
```

Comment:
nit: why is the visibility modifier after the type name? doesn't it usually prefix the type declaration?